packages/flex-plugin-scripts/src/scripts/start.ts

```diff
@@ -165,7 +181,58 @@ export const _updatePluginsUrl = (port: number) => {
 
   // Replace port and re-write to file
   plugins[pluginIndex].src = plugins[pluginIndex].src.replace(matches[1], port.toString());
-  fs.writeFileSync(paths().app.pluginsJsonPath, JSON.stringify(plugins, null, 2));
+  writeJSONFile(getPaths().app.pluginsJsonPath, plugins);
+};
+
+/**
+ * Update port of a plugin
+ * @param port
+ * @param name
+ */
+export const _updatePluginPort = (port: number, name: string) => {
+  const config = readPluginsJson();
+  config.plugins.forEach((plugin) => {
+    if (plugin.name === name) {
+      plugin.port = port;
+    }
+  });
+
+  writeJSONFile(getPaths().cli.pluginsJsonPath, config);
+};
+
+/**
+ * Reads user input to returns the --name plugins
```

Comment:
Can you extend the docblock here and explain all the different formats of the `--name` that can be provided and what happens with each of those formats?